src/components/editStudentRow/EditStudentRow.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import TableCell from "@mui/material/TableCell";
+import TableRow from "@mui/material/TableRow";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+
+interface EditStudentRowProps {
+  label: string;
+  value: string | number;
+  handleInputChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+export default function EditStudentRow({
+  label,
+  value,
+  handleInputChange,
+}: EditStudentRowProps) {
+  return (
+    <Table>
```

Comment:
I personally wouldn't use a table here, but if you want to I'd have a single `<Table/>` wrapper instead of having one per row

Reply:
I refactored student_id.tsx to the form with uncontrolled inputs technique, so I deleted the EditStudentRow folder.